use-timescale/data-tiering/disabling-data-tiering.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Creating a tiering policy
+excerpt: How to create a tiering policy
+product: [ cloud ]
+keywords: [ tiered storage, tiering ]
+tags: [ storage, data management ]
+---
+
+# Disabling tiering on a hypertable
+
+If you no longer want to use tiered storage for a particular hypertable, you 
+can drop the associated metadata by calling the `disable_tiering` function.
+
+In order to use this API, first call [remove_tiering_policy][tiering-policy] to drop 
+any tiering policy associated with this hypertable. Second, make sure that there is 
+no tiered data associated with this hypertable.
+
+This query lists the tiered chunks associated with this hypertable.
+```sql
+select * from timescaledb_osm.tiered_chunks 
+```
+
+If you have any tiered chunks, you must either untier this data or drop these chunks 
```

Comment:
@gayyappan you may want to disable tiering on a service but leave the tiered data - for example, if there is another service (fork, etc) that has access to the same data. Am I right?

Reply:
This is metadata specific to this service/instance. It does not control what happens on a fork.

Reply:
We added this API in response to users who want to untier everything and then make some schema modifications. We had to manually go and fix everything to remove tiering capabilities for that hypertable - drop the foreign table etc. 
Or they still foreign chunks in their plans etc.

Probably needs a rewrite if this is confusing. Please take a stab at it.

Reply:
IMO, Bringing up another service in this context is confusing. A fork always respects a fork's semantics (i.e. the  primary and fork will/can diverge).
a replica respects a replica's semantics i.e. primary and replica will be in sync.
So this is not related to tiering?

Reply:
I made some changes to the doc. Hopefully this reads better now. @billy-the-fish  @iroussos all your now :-)